engine/src/main/kotlin/world/gregs/voidps/engine/client/ui/SongEndEvent.kt

```diff
@@ -0,0 +1,24 @@
+package world.gregs.voidps.engine.client.ui
+
+import world.gregs.voidps.engine.entity.character.player.Player
+import world.gregs.voidps.engine.event.Event
+import world.gregs.voidps.engine.event.EventDispatcher
+import world.gregs.voidps.engine.event.Events
+
+data class SongEndEvent(
+    val songIndex: Int
+) : Event {
+    override val size = 2
+
+    override fun parameter(dispatcher: EventDispatcher, index: Int): Any? {
+        return when(index) {
+            0 -> "song_end"
+            1 -> songIndex
```

Comment:
Doesn't need to be changed but as the handler isn't matching against songIndex, index 1 isn't needed here